contracts/StablecoinConverter.sol

```diff
@@ -249,6 +255,22 @@ contract StablecoinConverter is EpochTokenLocker {
         );
     }
 
+    function evaluateDisregardedUtility(Order memory order, address user) internal view returns(uint128) {
+        // |disregardedUtility| = (limitTerm * leftoverSellAmount) / order.sellAmount
+        // where limitTerm = price.SellToken * order.sellAmt - order.buyAmt * price.buyToken
```

Comment:
Could you maybe explain what this "limitTerm" is supposed to represent?

Reply:
Yes certainly! But only after I fix the failing utility evaluation test!